lib/gui.py

```diff
@@ -96,18 +96,101 @@ def on_submit():
     submit_button.grid(row=3, column=1)
 
 
+def make_port_scan(main_window):
+    font = ""
+    width = 800
+    height = 600
+    port_scan_window = tk.Toplevel(main_window, bg="#222", width=width, height=height)
+    port_scan_window.resizable(False, False)
+    port_scan_window.title("Port Scanner")
+
+    port_scan_frame = tk.Frame(port_scan_window)
+    port_scan_frame.pack(fill=tk.BOTH, expand=True)
+
+    host = tk.StringVar()
+    host.set("127.0.0.1")
```

Comment:
We could set the value directly, avoiding this `.set()` call like so:
```python
host = tk.StringVar(value="127.0.0.1")
```
This applies to other similar cases as well

Reply:
changed